### ISSUE TITLE
VariableForce: undefined method `conditional?' for nil:NilClass

### ISSUE THREAD

**Issue by yskkin** [rubocop/rubocop#10899]
## Expected behavior

Exit without error.

## Actual behavior

```
An error occurred while VariableForce cop was inspecting /path/to/test.rb.
undefined method `conditional?' for nil:NilClass
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/lint/shadowing_outer_local_variable.rb:73:in `same_conditions_node_different_branch?'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/lint/shadowing_outer_local_variable.rb:60:in `before_declaring_variable'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:34:in `public_send'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:34:in `block in run_hook'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:31:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:31:in `run_hook'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:381:in `block (2 levels) in <class:VariableForce>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force/variable_table.rb:17:in `invoke_hook'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force/variable_table.rb:50:in `declare_variable'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:149:in `process_variable_declaration'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:87:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:95:in `inspect_variables_in_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:284:in `process_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:87:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:163:in `process_variable_assignment'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:95:in `inspect_variables_in_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:284:in `process_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:80:in `investigate'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `block (2 levels) in invoke'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:160:in `with_cop_error_handling'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `block in invoke'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `invoke'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:85:in `investigate'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/team.rb:155:in `investigate_partial'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/team.rb:83:in `investigate'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:311:in `inspect_file'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:255:in `block in do_inspection_loop'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:289:in `block in iterate_until_no_changes'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:282:in `loop'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:282:in `iterate_until_no_changes'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:251:in `do_inspection_loop'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:134:in `block in file_offenses'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:159:in `file_offense_cache'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:133:in `file_offenses'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:69:in `block in warm_cache'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:515:in `call_with_index'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:485:in `process_incoming_jobs'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:465:in `block in worker'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:456:in `fork'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:456:in `worker'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:447:in `block in create_workers'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:446:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:446:in `each_with_index'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:446:in `create_workers'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:386:in `work_in_processes'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:289:in `map'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/parallel-1.21.0/lib/parallel.rb:238:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:69:in `warm_cache'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:46:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command.rb:11:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/environment.rb:18:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli.rb:72:in `run_command'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli.rb:79:in `execute_runners'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli.rb:48:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/exe/rubocop:19:in `block in <top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/benchmark.rb:308:in `realtime'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/exe/rubocop:19:in `<top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/bin/rubocop:23:in `load'
/Users/xxxx/.rbenv/versions/2.7.6/bin/rubocop:23:in `<top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `load'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `kernel_load'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli/exec.rb:28:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli.rb:476:in `exec'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli.rb:30:in `dispatch'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli.rb:24:in `start'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/libexec/bundle:46:in `block in <top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/libexec/bundle:34:in `<top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/bin/bundle:23:in `load'
/Users/xxxx/.rbenv/versions/2.7.6/bin/bundle:23:in `<main>'
Inspecting 1 file
Scanning /path/to/test.rb
An error occurred while VariableForce cop was inspecting /path/to/test.rb.
undefined method `conditional?' for nil:NilClass
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/lint/shadowing_outer_local_variable.rb:73:in `same_conditions_node_different_branch?'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/lint/shadowing_outer_local_variable.rb:60:in `before_declaring_variable'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:34:in `public_send'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:34:in `block in run_hook'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:31:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/force.rb:31:in `run_hook'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:381:in `block (2 levels) in <class:VariableForce>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force/variable_table.rb:17:in `invoke_hook'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force/variable_table.rb:50:in `declare_variable'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:149:in `process_variable_declaration'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:87:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:95:in `inspect_variables_in_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:284:in `process_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:87:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:163:in `process_variable_assignment'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:103:in `block in process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:28:in `block in each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-ast-1.21.0/lib/rubocop/ast/node/mixin/descendence.rb:25:in `each_child_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:100:in `process_children'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:95:in `inspect_variables_in_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:284:in `process_scope'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:86:in `process_node'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/variable_force.rb:80:in `investigate'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `block (2 levels) in invoke'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:160:in `with_cop_error_handling'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `block in invoke'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:153:in `invoke'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/commissioner.rb:85:in `investigate'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/team.rb:155:in `investigate_partial'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cop/team.rb:83:in `investigate'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:311:in `inspect_file'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:255:in `block in do_inspection_loop'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:289:in `block in iterate_until_no_changes'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:282:in `loop'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:282:in `iterate_until_no_changes'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:251:in `do_inspection_loop'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:134:in `block in file_offenses'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:159:in `file_offense_cache'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:133:in `file_offenses'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:124:in `process_file'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:105:in `block in each_inspected_file'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:104:in `each'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:104:in `reduce'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:104:in `each_inspected_file'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:90:in `inspect_files'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/runner.rb:47:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/command.rb:11:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli/environment.rb:18:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli.rb:72:in `run_command'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli.rb:79:in `execute_runners'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/lib/rubocop/cli.rb:48:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/exe/rubocop:19:in `block in <top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/benchmark.rb:308:in `realtime'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.34.1/exe/rubocop:19:in `<top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/bin/rubocop:23:in `load'
/Users/xxxx/.rbenv/versions/2.7.6/bin/rubocop:23:in `<top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `load'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `kernel_load'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli/exec.rb:28:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli.rb:476:in `exec'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli.rb:30:in `dispatch'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/cli.rb:24:in `start'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/libexec/bundle:46:in `block in <top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/2.7.0/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/Users/xxxx/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/libexec/bundle:34:in `<top (required)>'
/Users/xxxx/.rbenv/versions/2.7.6/bin/bundle:23:in `load'
/Users/xxxx/.rbenv/versions/2.7.6/bin/bundle:23:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while VariableForce cop was inspecting /path/to/test.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.34.1 (using Parser 3.1.2.1, rubocop-ast 1.21.0, running on ruby 2.7.6 arm64-darwin20)
```

## Steps to reproduce the problem

Lint following content:

```ruby
def foo
  v = if c == 'x'
        xx { |v| yy(v) }
      end
end
```

## RuboCop version

1.34.1 (using Parser 3.1.2.1, rubocop-ast 1.21.0, running on ruby 2.7.6 arm64-darwin20)
